test/gurka/test_gtfs.cc

```diff
@@ -151,16 +151,14 @@ const gurka::ways ways = {{"AB", {{"highway", "primary"}}}, {"BC", {{"highway",
 
 boost::property_tree::ptree get_config() {
 
-  return test::make_config(VALHALLA_BUILD_DIR "test/data/transit_tests",
-                           {{"mjolnir.transit_feeds_dir",
-                             VALHALLA_BUILD_DIR "test/data/transit_tests/gtfs_feeds"},
-                            {"mjolnir.transit_dir",
-                             VALHALLA_BUILD_DIR "test/data/transit_tests/transit_tiles"},
+  return test::make_config("test/data/transit_tests",
+                           {{"mjolnir.transit_feeds_dir", "test/data/transit_tests/gtfs_feeds"},
+                            {"mjolnir.transit_dir", "test/data/transit_tests/transit_tiles"},
                             {"mjolnir.transit_pbf_limit",
                              "1"}, // so we create more than one file per tile
                             {"mjolnir.hierarchy", "1"},
-                            {"mjolnir.timezone", VALHALLA_BUILD_DIR "test/data/tz.sqlite"},
```

Comment:
hm, do we really want to remove it here? or maybe it's added somewhere else I didn't yet?

Reply:
yeah this is the primary open issue at this point. the signature for `make_config` takes first the directory argument and second a map of key values replacements, some of which might be things that are directories some not.

it was easy for me to patch the first arg when it didnt have the absolute path because i knew it was a directory but for the key values i dont know which ones are and arent directories. this means detecting and patching those is harder.

there are two options to do it automatically:

1. something simple where i see the value begins with `"test/data"` this will not catch everythign but it should get most of everything
2. figure out which ones should be directories by looking at the default config for the path value that got substituted in everywhere and then patching only those that were substituted paths

i think ill take a try at the 2nd one because then everything can be completely automatic and i can remove any of these that exist

Reply:
that'd be much appreciated, albeit quite a sizable mechanical chore